Makefile

```diff
@@ -222,14 +222,14 @@ help:
 	@echo ""
 	@echo "# build"
 	@echo ""
-	@echo "    $$ make all                      # build tracee-ebpf, tracee-rules & rules"
+	@echo "    $$ make all                      # build tracee-ebpf, tracee-rules & signatures"
 	@echo "    $$ make bpf-core                 # build ./dist/tracee.bpf.core.o"
 	@echo "    $$ make bpf-nocore               # build ./dist/tracee.bpf.XXX.o"
 	@echo "    $$ make tracee-ebpf              # build ./dist/tracee-ebpf"
 	@echo "    $$ make tracee-rules             # build ./dist/tracee-rules"
```

Comment:
I believe you are keeping tracee rules because it will get deprecated with the single binary. That might be the reason why you didn't change tracee-rules packaging, let me go back there =).

Reply:
👍